tests/modules/nf-scil/reconst/diffusivitypriors/nextflow.config

```diff
@@ -0,0 +1,11 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: "RECONST_DIFFUSIVITYPRIORS" {
```

Comment:
I'm not sure your parameters here apply to your tests. You are renaming both processes in your test cases. Please validate.

Reply:
I've just tested it. If you enter the module name, it applies the parameters even if you "rename it via as".
On the other hand, if you enter the name corresponding to as, it only applies to the renamed module. 

This can be useful if you want to run a module with different parameters. But if these are general parameters used all the time, the module name is better, isn't it?

In that case, I'll leave it as it is or just put COMPUTE_PRIORS. Which do you prefer?

Reply:
This is perfect ! I was not sure we had that behavior, I'm happy we do ! (written for posterity)